Task_1/Task_1/src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import Form from "./components/Input_form/form";
+import List from "./components/to_do_list/List";
+import { useState } from "react";
+import moment from 'moment'
+
+function App() {
+  console.log("_____ render_______");
+
+
+  let TaskList = [
+  ]
+
+  const [taskList, setTaskList] = useState(TaskList);
```

Comment:
instead of defining a variable then using in useState we can directly use useState([]).

Reply:
Updated